vme/zone/professions.zon

```diff
@@ -31,6 +31,45 @@ thief: assassin<br/>
 cleric: druid<br/>
 mage: sorcerer<br/>
 
+***** Changes to allow more profession variation ******
+
+You can now choose to specialize in your first or second profession as
+long as it is a basic profession. You can still only specialize once.
+Here are some examples
```

Comment:
Maybe add:
You can only learn one of the specialization classes from a given base class.

Reply:
Added more details